website/blog/2022-05-27-aluminium.md

```diff
@@ -0,0 +1,278 @@
+
+![override-completion](https://i.imgur.com/Go3sMxy.gif)
+
+## [Scala3] Show scaladoc on hover for Scala 3 project
+Now, Metals shows the scaladoc on hover for Scala3 projects. (Before this release, Metals was unable to show the scaladoc for the symbols from third-party modules).
+
+![hover-scala3](https://i.imgur.com/Svzq5DD.png)
+
+### [Scala3] Support `completionItem/resolve`
+Now, in Scala 3 we get scaladoc that shows the default values when moving across completions. 
+
+![completion-item-resolve](https://i.imgur.com/Tz6AOsx.gif)
+
+
+## Show parent scaladoc if implementation is returning empty
+
+Scaladocs can be inspected whenever you hover, use a completion or signature help. Up until recently we only showed you the documentation if the exact method you are using had the scaladocs written, which meant that if you overrode a method and didn't add the scaladoc comments again we would not show you any documentation. One example of such method is `headOption` on `List`.
+
+ From this release we will also search the parent method in case the current method's scaladoc are empty. 
+
+## [Scala 3] Show scaladocs for signature help
+
+As mentioned in the previous paragraph, Metals can show you documentation in three different places. That, however, was true only for Scala 2 previously. In this release, we will now show you proper documentation whenever invoking signature help.
+
+As a reminder, signature help is used to indicate what parameters can be used in a method. It should pop up automatically after writing `(`, but you can also invoke it manually. In VS Code that takes the form of `editor.action.triggerParameterHints` command, which can also be bound to a shortcut and by default is.
+
+## [MUnit] Test Explorer can find helper methods from parent classes
+
+MUnit allows to use [helper functions](https://scalameta.org/munit/docs/tests.html#declare-tests-inside-a-helper-function) when declaring tests. Very often those helper methods are extracted to some parent classes which are extended by many test suites.
+Now, Metals can find usages of those helper methods and display them in Test Explorer.
+
+This feature is available for Bloop and SBT 1.7.0-M2 or later.
+
+![MUnit-helper-methods](https://imgur.com/GGRDpXA.gif)
+
+## Support Cats Effect stacktraces in stacktrace analyzer
+
+Cats Effect offers [asynchronous stack tracing](https://typelevel.org/cats-effect/docs/tracing#asynchronous-stack-tracing) which augments exceptions with additional information.  
+Now, Stacktrace Analyzer is able to recognize CE's stacktraces and provide link to location in code. Say no to tedious debugging when you have only stacktrace from the logs!
+
+![cats-effect-stacktraces](https://imgur.com/5fMvcYd.gif)
+
+
+## Improve implement all completion and code action
+Previously, when we invoked the "implement all members" completion and code action, Metals used `x$0` for Java parameter names. Now, Metals fills these with the correct parameter names.
+ 
+
```

Comment:
```suggestion
## Improve rewrite to braces/parenthesis code action

Sometimes it wasn't clear which code would be affected by [rewrite to braces/parenthesis](https://scalameta.org/metals/blog/2021/09/06/tungsten#replace--with--in-functions-and-vice-versa) code action.
Now, code action's description contains name of the function/method which will be affected by executing action.

![rewrite-braces-parens](https://imgur.com/SkHolsJ.gif)
Theme: [One Dark Pro](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)

```

Reply:
I think it's handy to include theme name for each GIF that was included in release notes

Reply:
Thank you for adding words to a release note!

> I think it's handy to include the theme name for each GIF that was included in release notes

I personally think I don't need to add a theme name to release notes because most of the readers of this note are interested in what's new in the latest Metals. Color theme is sometimes interesting, but they are not related to the Metals.